docs/source/api/neuralhydrology.datasetzoo.camelsind.rst

```diff
@@ -0,0 +1,7 @@
+CamelsIND
+========
```

Comment:
I see the docs test failing. I assume this is because the underlining of the title has to be of the exact same lengths as the title. If I see correctly, there is one "=" missing.
```suggestion
CamelsIND
=========
```